libkbfs/root_metadata.go

```diff
@@ -286,6 +285,13 @@ func (md *RootMetadata) MakeSuccessor(
 		return nil, errors.New("MD with invalid revision")
 	}
 	newMd.SetRevision(md.Revision() + 1)
+
+	merkleSeqNo, err := merkleGetter.GetCurrentMerkleSeqNo(ctx)
+	if err != nil {
+		return nil, err
+	}
+	newMd.SetMerkleSeqNo(merkleSeqNo)
```

Comment:
No, the sequence number is increased according to the server-side global Merkle tree updating.  The client can't update it -- from the KBFS client's perspective, it's just a monotonically-increasing magic number given to it from the service.